README.md

```diff
@@ -45,26 +33,6 @@ For easy testing, they are combined in:
 
 But the above 4 files are always leading.
 
-The files:
-
-- examples/*.json
-
-are examples created in vitro & very artificial.
-
-
 ## Implementation Notes
 
 A list of [Frequently Asked Questions](https://github.com/ehn-digital-green-development/ehn-dgc-schema/wiki/FAQ) for implementors can be found in the [repository wiki](https://github.com/ehn-digital-green-development/ehn-dgc-schema/wiki).
```

Comment:
Old URLs for this repo.

Reply:
yep - because the sudden name change could have been a lot better managed, feel free to fix plus any others you find

Reply:
Will do. Also: GitHub handles the rename gracefully with a redirect, so no show-stopper anyway.

Reply:
Yeah is a re-direct, but all the same the text should be corrected to point to new.
Fixed in #b84ccf4 - thx for that.